content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -496,3 +496,24 @@ Then, simply bind both `VirtualServices` to it like this:
 - `Gateway` configuration `gw` with host `*.test.com`, selector `istio: ingressgateway`, and TLS using gateway's mounted (wildcard) certificate
 - `VirtualService` configuration `vs1` with host `service1.test.com` and gateway `gw`
 - `VirtualService` configuration `vs2` with host `service2.test.com` and gateway `gw`
+
+### Configuring SNI routing when not sending SNI
+
+An HTTPS `Gateway` configuring the `hosts` field will perform an [SNI](https://en.wikipedia.org/wiki/Server_Name_Indication) match on incoming requests.
+For example, the following configuration would only allow requests that match `*.example.com` in the SNI:
+
+{{< text yaml >}}
+servers:
+- port:
+    number: 443
+    name: https
+    protocol: HTTPS
+  hosts:
+  - "*.example.com"
+{{< /text >}}
+
+This may cause certain requests to fail.
+
+For example, if you do not have DNS set up and are instead directly setting the host header, such as `curl 1.2.3.4 -H "Host: app.example.com"`, no SNI will be set, causing the request to fail. Instead, you can set up DNS or use the `--resolve` flag of `curl`.
```

Comment:
By "do not have DNS set up" I assume you mean that your domain name can be looked up publicly? Maybe clarify the text a bit and add a link.

Also, it would be good to explain what the `--resolve` flag does and how it fixes the issue.

Reply:
It might still be a good idea, but this page uses `--resolve` many times already. But yes that is what I meant about DNS, I wasn't sure a good way to phrase it.